src/components/NftPreview/NftPreview.tsx

```diff
@@ -200,13 +155,12 @@ const StyledNftPreview = styled.div<{
   justify-content: center;
   align-items: center;
   position: relative;
-  height: fit-content;
   overflow: hidden;
 
   ${({ backgroundColorOverride }) =>
     backgroundColorOverride && `background-color: ${backgroundColorOverride}`}};
 
-  max-height: 80vh;
+
```

Comment:
i think this used to make sure that portrait NFTs look, but can take a look

Reply:
ah yup definitely. def don't want to remove, good catch
now that I can point to dev or prod, I'll do more testing with different tokens